samples/saga/simple/Core_6/Sample/Program.cs

```diff
@@ -16,23 +14,14 @@ static void Main()
     {
         Console.Title = "Samples.SimpleSaga";
         var endpointConfiguration = new EndpointConfiguration("Samples.SimpleSaga");
-        endpointConfiguration.UseSerialization<JsonSerializer>();
-        endpointConfiguration.EnableInstallers();
-
-        var recoverability = endpointConfiguration.Recoverability();
-        recoverability.DisableLegacyRetriesSatellite();
```

Comment:
without this the learning transport creates a redundant `Retries` directory structure